Dockerfile

```diff
@@ -30,5 +33,9 @@ COPY --chown=ubuntu:ubuntu ./rcsb /app/rcsb
 
 USER ubuntu
 
+# Workaround to mount OpenEye license secret into runner container
+RUN mkdir ~/oe-license && \
+    ln -s ~/.secrets/oe-license.txt ~/oe-license/oe-license.txt
+
```

Comment:
Since the `~/oe-license` directory doesn't exist already, we should be able to mount the secrets as a volume underneath that path and avoid using a symlink workaround here.